test/e2e/common/node/podtemplates.go

```diff
@@ -159,6 +161,44 @@ var _ = SIGDescribe("PodTemplates", func() {
 
 		err = wait.PollImmediate(podTemplateRetryPeriod, podTemplateRetryTimeout, checkPodTemplateListQuantity(f, "podtemplate-set=true", 0))
 		framework.ExpectNoError(err, "failed to count required pod templates")
+
+	})
+
+	ginkgo.It("should replace a pod template", func() {
+		ptClient := f.ClientSet.CoreV1().PodTemplates(f.Namespace.Name)
+		ptName := "podtemplate-" + utilrand.String(5)
+
+		ginkgo.By("Create a pod template")
+		ptResource, err := ptClient.Create(context.TODO(), &v1.PodTemplate{
```

Comment:
I don't see other tests doing it, but it may be a good practice to clean up objects created by the test. wdyt?

Reply:
Everything in the test namespace are all deleted by the framework when the test run finishes.